qa/tasks/lost_unfound.py

```diff
@@ -7,14 +7,15 @@
 from tasks.util.rados import rados
 from teuthology import misc as teuthology
 from teuthology.orchestra import run
+#import json
 
 log = logging.getLogger(__name__)
 
 def task(ctx, config):
     """
     Test handling of lost objects.
 
-    A pretty rigid cluseter is brought up andtested by this task
+    A pretty rigid cluseter is brought up and tested by this task
```

Comment:
nit: could you fix the typo in `cluster` too

Reply:
I see this missing space in `ec_lost_unfound.py` and `rep_lost_unfound_delete.py`. What does `cluster` mean?

Reply:
Fixed cluster